### PR TITLE
Doc: Added tutorial documentation for stats distributions ksone and kstwobign

### PR DESCRIPTION
**Pull request by pvanmulbregt**
The stats tutorial pages in the doc for these two KS distributions existed but were empty. This adds a body for each of them.

### CODE REVIEW

**Review by rgommers — CHANGES_REQUESTED** [review]
Thanks @pvanmulbregt! Had a few minor comments only so far. ``ksone`` seems right to me, ``kstwobign`` is hard to check - not in the Birnbaum paper. Did you have another reference for that?

File: doc/source/tutorial/stats/continuous_ksone.rst [hunk 1]
```diff
@@ -4,4 +4,39 @@
...
+empirical distribution function, computed from n samples or observations,
+and a comparison (or target) cumulative distribution function.
+
+Writing :math:`D_n^+ = \sup_t \left(F_{target}(t)-F_{empirical,n}(t)\right)`,
+KSone is the distribution of the :math:`D_n^+` values.
+(The distribution of :math:`D_n^- = -\inf_t \left(F_{target}(t)-F_{empirical,n}(t)\right)`
+ differences follows the same distribution, so ksone can be used for one-sided tests on either side.)
+
+
+One shape parameter
```
Comment (pvanmulbregt) [inline doc/source/tutorial/stats/continuous_ksone.rst]: Requested changes now made.
I had cloned the style from the beta distribution doc, tutorial/stats/continuous_beta.html.  Is there a preferred template one should use?

Comment (rgommers) [inline doc/source/tutorial/stats/continuous_ksone.rst]: Not really I think. Most of these have been converted to Sphinx from LaTeX a long time ago, that's why the format looks fairly uniform. But I'm not sure I can point to a template or one function as the preferred one.

**Comment by rgommers** [comment]
https://2976-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/tutorial/stats/continuous_ksone.html

https://2976-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/tutorial/stats/continuous_kstwobign.html

both look good. change to ``target`` and ``empirical`` made it clearer

**Comment by rgommers** [comment]
Merged, thanks @pvanmulbregt

**Pull request merged**